app/demo.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App;
+
+use GFAPI;
+use GFExport;
+use GFCommon;
+
+/**
+ * Create a demo Gravityform when the theme is activated.
+ */
+add_action('after_switch_theme', function () {
+    if (!class_exists('GFAPI')) {
+        return;
+    }
+
+    if (!empty(GFAPI::get_forms())) {
+        return;
+    }
+
+    require_once GFCommon::get_base_path() . '/export.php';
+    GFExport::import_file(get_stylesheet_directory() . '/resources/demo/gravityforms.json');
+});
```

Comment:
Ok, so when activating the theme, it will create a sample form.
Is this also included in the gutenberg demo?

Reply:
Yes